src/UglyToad.PdfPig.SystemDrawing/SystemDrawingProcessorV2.cs

```diff
@@ -0,0 +1,1287 @@
+﻿namespace UglyToad.PdfPig.SystemDrawing
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Diagnostics;
+    using System.Drawing;
+    using System.Drawing.Drawing2D;
+    using System.Drawing.Imaging;
+    using System.Drawing.Text;
+    using System.IO;
+    using System.Linq;
+    using System.Runtime.InteropServices;
+    using UglyToad.PdfPig.Content;
+    using UglyToad.PdfPig.Core;
+    using UglyToad.PdfPig.Graphics;
+    using UglyToad.PdfPig.Graphics.Colors;
+    using UglyToad.PdfPig.Graphics.Core;
+    using UglyToad.PdfPig.Graphics.Shading;
+    using UglyToad.PdfPig.PdfFonts;
+    using UglyToad.PdfPig.Tokens;
+    using UglyToad.PdfPig.XObjects;
+
+    public class SystemDrawingProcessorV2 : IOperationContext, IDrawingProcessor
```

Comment:
I'm assuming this is still work in progress, but the `V2` makes me unhappy, but it'll do for now :D

Reply:
yes indeed, it's wip so no need to worry 😄